### ISSUE TITLE
Set Title Attribute

### ISSUE THREAD

**Issue by AdamTheRedBaron** [beaverbuilder/feature-requests#86]
There is currently nowhere to set a title attribute for links. This is an HTML attribute and cannot be set using a css class. 

It would be good to have another check ☑ box next to the "Open in new window" and "Nofollow" boxes which opens a text field where this could be entered. 

Alternatively, just a text field.

**Comment by zackpyle** [comment]
I think it would need to be a repeater of sorts, like is has been done here so multiple can be added: https://github.com/JasonTheAdams/BBCustomAttributes

Two questions here as well. Would it be placed on the base element and not the containing div? That is the problem with this current solution: https://docs.wpbeaverbuilder.com/beaver-builder/developer/how-to-tips/add-custom-attributes-to-rows-columns-or-modules/
The attribute should be on the element itself (ex: h2, img, etc)

**Comment by zackpyle** [comment]
@deargeek Agreed, but until then, you can use [this plugin](https://github.com/JasonTheAdams/BBCustomAttributes) to add whatever attributes you need?

**Comment by deargeek** [comment]
@zackpyle I have tried it out, but with no impact on PSI. But, this could relate to some WP Rocket settings, so it would be nicer to do it natively.

**Comment by zackpyle** [comment]
@deargeek Yes, I agree it should be native, but if you _do_ want to use it, it is possible to do what you're asking.

If you are delaying all js with WP Rocket, you should make an exclusion for` bb-custom-attributes.js`

The js file is less than 1kb (it's like 900 bytes) so it's tiny but will fix your scores.

**Comment by deargeek** [comment]
@zackpyle thanks, I'll keep playing.
